components/ActiveLink/ActiveLink.tsx

```diff
@@ -17,7 +17,7 @@ const ActiveLink = ({
   const className = asPath === rest.href ? activeClassName : "";
 
   return (
-    <Link {...rest}>
+    <Link legacyBehavior {...rest}>
```

Comment:
Why do we need this here?
> An <a> element is no longer required as a child of <Link>. Add the legacyBehavior prop to use the legacy behavior or remove the <a> to upgrade

Is the anchor tag generated by `cloneElement` or something?

Reply:
It's a compatibility issue related to upgrading to Next13, it says that <Link> has a <a> child element and that it can't. It tells me to remove it or add that flag so it works as before. It won't compile without that.